lambdas/models/bulk_upload_status.py

```diff
@@ -34,6 +35,7 @@ class FailedUpload(UploadStatusBaseClass):
     "Date",
     "Timestamp",
     "ID",
+    "OdsCode",
```

Comment:
Just a trivial thing, but could we change the order of this list and move "OdsCode" higher? For example after the FailureReason. 
The order of this list will become the column order of report from left to right, so this could help people who follow up the "Patient not registered at your practice" failure